Fix menu price

armandohimself opened the pull request:
Just finished the Menus. Went through everything with my changes and everything is looking really good so far. Tell me what you think.

armandohimself committed:
Added Tooltip to Menu Pricing

armandohimself committed:
Finished Menu Pricing

- You can create a menu from scratch and suggested pricing does not show up. 
- You can create a menu from a template and the suggested pricing shows any time you click on a a templated menu in My Menus
- Added small styling changes that will help the view pop a bit more with the card headers/lables.

JoshuaArcyLHarrison commented on versions/app/src/styles/styles.js:
```diff
@@ -183,7 +188,7 @@ const globalStyles =  StyleSheet.create({
     navigate_away_content:{
         flex:1,
-        fontWeight: 'bold',
+        fontWeight: 'medium',
```
This throws an error:
> Invalid prop `fontWeight` of value `medium` supplied to `Text`, expected one of ["normal","bold","100","200","300","400","500","600","700","800","900"].

See which one does what you want it to.

armandohimself replied:
Thank you for catching this!

JoshuaArcyLHarrison commented:
Suggested edit:
```diff
diff --git a/versions/app/src/styles/styles.js b/versions/app/src/styles/styles.js
index d8eda87..575fc07 100644
--- a/versions/app/src/styles/styles.js
+++ b/versions/app/src/styles/styles.js
@@ -1,390 +1,385 @@
-import { StyleSheet } from 'react-native';
-import Theme from '../styles/theme.style.js';
-
-const globalStyles =  StyleSheet.create({
-    safe_dark: {
-        flex:1,
-        backgroundColor: Theme.PRIMARY_COLOR
-    },
-    safe_light: {
-        flex:1,
-        backgroundColor: Theme.BACKGROUND_COLOR,
-        padding:0,
-        margin:0,
-    },
-    safe_trans: {
-        flex:1,
-    },
-    scrollContainer: {
-        flex: 1,
-    },
-    page: {
-        flex: 1,
-        backgroundColor: Theme.BACKGROUND_COLOR,
-        alignItems: 'center',
-        justifyContent:'flex-start',
-        padding: 10,
-    },
-    page_centered: {
-        flex: 1,
-        alignItems: 'center',
-        justifyContent:'center',
-        padding: 10,
-    },
-    page_top: {
-        flex: 1,
-        alignItems: 'stretch',
-        justifyContent:'flex-start',
-        padding: 10,
-    },
-    page_bottom: {
-        flex: 1,
-        alignItems: 'stretch',
-        justifyContent:'flex-end',
-        padding: 10,
-    },
-    container: {
-        alignItems: 'center',
-        justifyContent:'flex-start',
-    },
-    h1: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontWeight: Theme.FONT_WEIGHT_MEDIUM,
-        fontSize: Theme.FONT_SIZE_XL,
-        color:Theme.TEXT_ON_SURFACE_COLOR,
-        lineHeight: 33,
-        paddingVertical:10,
-    },
-    h2: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_LARGE,
-        color:Theme.TEXT_ON_SURFACE_COLOR,
-        lineHeight: 20,
-        paddingVertical:8,
-    },
-    h3: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_MEDIUM_PLUSM,
-        fontWeight: 'bold',
-        color:Theme.TEXT_ON_SURFACE_COLOR,
-        lineHeight: 16,
-        paddingVertical:6,
-    },
-    subtitle: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_SMALL,
-        fontSize:13,
-        lineHeight:16,
-        color: Theme.TEXT_ON_SURFACE_COLOR_LIGHT
-    },
-    p: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_SMALL,
-        color:Theme.TEXT_ON_SURFACE_COLOR,
-        fontSize: 14,
-        lineHeight: 20,
-    },
-    card: {
-        paddingVertical:15,
-        paddingHorizontal:12,
-        borderRadius: 8,
-        backgroundColor: Theme.SURFACE_COLOR,
-        margin: 5,
-        shadowColor: Theme.FAINT,
-        shadowOffset: {
-            width: 2,
-            height: 2,
-        },
-        shadowOpacity: .4,
-        shadowRadius: 3,
-        elevation: 3,
-    },
-    card_header: {
-        flexDirection: "row",
-        justifyContent: "space-between",
-        marginBottom: 7,
-    },
-    card_header_text: {
-        fontWeight: "bold",
-        fontSize: 16, 
-    },
-    divider: {
-        borderBottomColor: Theme.FAINT,
-        borderBottomWidth: 1,
-        paddingBottom: 15
-    },
-    list_cont: {
-        flexDirection: 'row',
-        alignItems: 'center',
-        marginVertical: 10
-    },
-    list_bullet: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_SMALL,
-        color: Theme.TEXT_ON_SURFACE_COLOR_LIGHT,
-        fontSize: 26,
-        lineHeight: 30,
-        paddingLeft: 5,
-    },
-    list_item: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_SMALL,
-        color: Theme.TEXT_ON_SURFACE_COLOR_LIGHT,
-        fontSize: 13,
-        lineHeight: 16,
-        paddingLeft: 8,
-    },
-    blurb_text_large: {
-        fontSize: Theme.FONT_SIZE_MEDIUM,
-        textAlign: 'center'
-    },
-    italic: {
-        fontStyle: 'italic'
-    },
-    badge_small: {
-        padding: 10,
-        marginHorizontal: 5,
-        marginVertical: 1,
-        width: undefined,
-        height: undefined,
-        resizeMode: 'contain'
-    },
-    badge_large: {
-        padding: 25,
-        marginHorizontal: 5,
-        marginVertical: 1,
-        width: undefined,
-        height: undefined,
-        resizeMode: 'contain',
-    },
-    empty_state:{
-        flex:1,
-        width:'100%',
-        justifyContent: 'center', 
-        alignItems: 'center'
-    },
-    empty_image: {
-        height: '40%',
-        resizeMode: 'contain',
-        margin: 0,
-        padding: 0
-    },
-    empty_text: {
-        fontSize: 13,
-        fontWeight: 'bold',
-        textAlign: 'center',
-        padding:20,
-        color: Theme.GRAY
-    },
-    navigate_away: {
-        width: '100%',
-        flexDirection:'row',
-        justifyContent: 'space-between',
-        alignItems: 'center',
-        padding: 15,
-        paddingHorizontal: 15,
-        borderBottomWidth:1,
-        borderColor: Theme.BORDER_COLOR
-    },
-    navigate_away_content:{
-        flex:1,
-        fontWeight: 'medium',
-        color: Theme.PRIMARY_COLOR,
-        fontSize: 13,
-    },
-    linkTextColor: {
-        color: Theme.SECONDARY_COLOR,
-    },
-})
-
+import { StyleSheet } from "react-native";
+import Theme from "../styles/theme.style.js";
 
+const globalStyles = StyleSheet.create({
+	safe_dark: {
+		flex: 1,
+		backgroundColor: Theme.PRIMARY_COLOR,
+	},
+	safe_light: {
+		flex: 1,
+		backgroundColor: Theme.BACKGROUND_COLOR,
+		padding: 0,
+		margin: 0,
+	},
+	safe_trans: {
+		flex: 1,
+	},
+	scrollContainer: {
+		flex: 1,
+	},
+	page: {
+		flex: 1,
+		backgroundColor: Theme.BACKGROUND_COLOR,
+		alignItems: "center",
+		justifyContent: "flex-start",
+		padding: 10,
+	},
+	page_centered: {
+		flex: 1,
+		alignItems: "center",
+		justifyContent: "center",
+		padding: 10,
+	},
+	page_top: {
+		flex: 1,
+		alignItems: "stretch",
+		justifyContent: "flex-start",
+		padding: 10,
+	},
+	page_bottom: {
+		flex: 1,
+		alignItems: "stretch",
+		justifyContent: "flex-end",
+		padding: 10,
+	},
+	container: {
+		alignItems: "center",
+		justifyContent: "flex-start",
+	},
+	h1: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontWeight: Theme.FONT_WEIGHT_MEDIUM,
+		fontSize: Theme.FONT_SIZE_XL,
+		color: Theme.TEXT_ON_SURFACE_COLOR,
+		lineHeight: 33,
+		paddingVertical: 10,
+	},
+	h2: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_LARGE,
+		color: Theme.TEXT_ON_SURFACE_COLOR,
+		lineHeight: 20,
+		paddingVertical: 8,
+	},
+	h3: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_MEDIUM_PLUSM,
+		fontWeight: "bold",
+		color: Theme.TEXT_ON_SURFACE_COLOR,
+		lineHeight: 16,
+		paddingVertical: 6,
+	},
+	subtitle: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_SMALL,
+		fontSize: 13,
+		lineHeight: 16,
+		color: Theme.TEXT_ON_SURFACE_COLOR_LIGHT,
+	},
+	p: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_SMALL,
+		color: Theme.TEXT_ON_SURFACE_COLOR,
+		fontSize: 14,
+		lineHeight: 20,
+	},
+	card: {
+		paddingVertical: 15,
+		paddingHorizontal: 12,
+		borderRadius: 8,
+		backgroundColor: Theme.SURFACE_COLOR,
+		margin: 5,
+		shadowColor: Theme.FAINT,
+		shadowOffset: {
+			width: 2,
+			height: 2,
+		},
+		shadowOpacity: 0.4,
+		shadowRadius: 3,
+		elevation: 3,
+	},
+	card_header: {
+		flexDirection: "row",
+		justifyContent: "space-between",
+		marginBottom: 7,
+	},
+	card_header_text: {
+		fontWeight: "bold",
+		fontSize: 16,
+	},
+	divider: {
+		borderBottomColor: Theme.FAINT,
+		borderBottomWidth: 1,
+		paddingBottom: 15,
+	},
+	list_cont: {
+		flexDirection: "row",
+		alignItems: "center",
+		marginVertical: 10,
+	},
+	list_bullet: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_SMALL,
+		color: Theme.TEXT_ON_SURFACE_COLOR_LIGHT,
+		fontSize: 26,
+		lineHeight: 30,
+		paddingLeft: 5,
+	},
+	list_item: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_SMALL,
+		color: Theme.TEXT_ON_SURFACE_COLOR_LIGHT,
+		fontSize: 13,
+		lineHeight: 16,
+		paddingLeft: 8,
+	},
+	blurb_text_large: {
+		fontSize: Theme.FONT_SIZE_MEDIUM,
+		textAlign: "center",
+	},
+	italic: {
+		fontStyle: "italic",
+	},
+	badge_small: {
+		padding: 10,
+		marginHorizontal: 5,
+		marginVertical: 1,
+		width: undefined,
+		height: undefined,
+		resizeMode: "contain",
+	},
+	badge_large: {
+		padding: 25,
+		marginHorizontal: 5,
+		marginVertical: 1,
+		width: undefined,
+		height: undefined,
+		resizeMode: "contain",
+	},
+	empty_state: {
+		flex: 1,
+		width: "100%",
+		justifyContent: "center",
+		alignItems: "center",
+	},
+	empty_image: {
+		height: "40%",
+		resizeMode: "contain",
+		margin: 0,
+		padding: 0,
+	},
+	empty_text: {
+		fontSize: 13,
+		fontWeight: "bold",
+		textAlign: "center",
+		padding: 20,
+		color: Theme.GRAY,
+	},
+	navigate_away: {
+		width: "100%",
+		flexDirection: "row",
+		justifyContent: "space-between",
+		alignItems: "center",
+		padding: 15,
+		paddingHorizontal: 15,
+		borderBottomWidth: 1,
+		borderColor: Theme.BORDER_COLOR,
+	},
+	navigate_away_content: {
+		flex: 1,
+		fontWeight: "500",
+		color: Theme.PRIMARY_COLOR,
+		fontSize: 13,
+	},
+	linkTextColor: {
+		color: Theme.SECONDARY_COLOR,
+	},
+});
 
 const menusStyles = StyleSheet.create({
-    menu_course_cont: {
-        marginBottom:20
-    },
-    menu_course: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_SMALL,
-        color:Theme.FAINT,
-        fontStyle: 'italic',
-        fontSize: 18,
-        lineHeight: 20,
-        textAlign: 'center',
-        paddingBottom: 10
-    },
-    menu_item_cont: {
-        paddingVertical:10,
-        marginBottom:10
-    },
-    menu_name: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_SMALL,
-        color:Theme.TEXT_ON_SURFACE_COLOR,
-        fontSize: 15,
-        lineHeight: 20,
-        fontWeight:'bold',
-        textAlign: 'center',
-        marginBottom:2
-    },
-    menu_desc: {
-        fontFamily: Theme.FONT_STANDARD,
-        fontSize: Theme.FONT_SIZE_SMALL,
-        color:Theme.TEXT_ON_SURFACE_COLOR,
-        fontSize: 14,
-        lineHeight: 20,
-        textAlign: 'center'
-    },
-})
+	menu_course_cont: {
+		marginBottom: 20,
+	},
+	menu_course: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_SMALL,
+		color: Theme.FAINT,
+		fontStyle: "italic",
+		fontSize: 18,
+		lineHeight: 20,
+		textAlign: "center",
+		paddingBottom: 10,
+	},
+	menu_item_cont: {
+		paddingVertical: 10,
+		marginBottom: 10,
+	},
+	menu_name: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_SMALL,
+		color: Theme.TEXT_ON_SURFACE_COLOR,
+		fontSize: 15,
+		lineHeight: 20,
+		fontWeight: "bold",
+		textAlign: "center",
+		marginBottom: 2,
+	},
+	menu_desc: {
+		fontFamily: Theme.FONT_STANDARD,
+		fontSize: Theme.FONT_SIZE_SMALL,
+		color: Theme.TEXT_ON_SURFACE_COLOR,
+		fontSize: 14,
+		lineHeight: 20,
+		textAlign: "center",
+	},
+});
 
 const modal = StyleSheet.create({
-    modalBackground: {
-        flex: 1,
-        justifyContent: "center",
-        alignItems: "center",
-        backgroundColor: 'rgba(12,30,52, 0.9)'
-    },
-    modalView: {
-        margin: 20,
-        backgroundColor: Theme.SURFACE_COLOR,
-        justifyContent: 'flex-start',
-        borderRadius: 10,
-        padding: 15,
-        alignItems: "center",
-        shadowColor: "#000",
-        shadowOffset: {
-            width: 0,
-            height: 2
-        },
-        shadowOpacity: 0.25,
-        shadowRadius: 4,
-        width: '90%',
-        height:420,
-        borderWidth:1,
-        borderColor: Theme.WHITE
-    },
-    close_button: {
-        position: 'absolute',
-        top: 0,
-        right:0,
-        padding: 10,
-        zIndex:10000,
-        fontWeight:Theme.FONT_WEIGHT_HEAVY
-    },
-    modalHeader: {
-        alignItems: 'center',
-        width: '100%',
-        color:Theme.SECONDARY_COLOR,
-    }
-})
-
-
-const forms =  StyleSheet.create({
-    formContainer: {
-        flexDirection: 'row',
-        height: 80,
-        marginTop: 40,
-        marginBottom: 20,
-        flex: 1,
-        paddingTop: 10,
-        paddingBottom: 10,
-        paddingLeft: 30,
-        paddingRight: 30,
-        justifyContent: 'center',
-        alignItems: 'center'
-    },
-    input: {
-        width: '100%',
-        fontSize:15,
-        lineHeight:17,
-        backgroundColor: Theme.SURFACE_COLOR,
-        color:Theme.TEXT_ON_SURFACE_COLOR,
-        marginVertical:15,
-        padding:10,
-        borderRadius: 5,
-        borderColor: Theme.FAINT,
-        overflow: 'hidden',
-        borderWidth:1,
-        backgroundColor: Theme.SURFACE_COLOR,
-        borderColor: Theme.BORDER_COLOR
-    },
-    textarea: {
-        height: 120,
-    },
-    search_bar: {
-        height: 48,
-        borderRadius: 25,
-        overflow: 'hidden',
-        backgroundColor: Theme.SURFACE_COLOR,
-        color:Theme.SECONDARY_COLOR,
-        borderWidth:1,
-        borderColor: Theme.PRIMARY_COLOR,
-        paddingLeft: 16,
-    },
-    placeholder_on_dark: {
-        color: 'rgba(203, 165, 44, 0.4)'
-    },
-    input_container: {
-        width: '100%',
-        flexDirection: 'row',
-        justifyContent: 'center',
-        alignItems: 'center',
-        paddingLeft:8,
-        height: 58,
-        borderRadius: 8,
-        marginVertical: 20,
-        borderWidth:1,
-        backgroundColor: Theme.SURFACE_COLOR,
-        borderColor: Theme.BORDER_COLOR
-    },
-    input_icon: {
-        color: Theme.SECONDARY_COLOR,
-        padding:8,
-        alignItems: 'center',
-    },
-    focused_dark:{
-        color: Theme.WHITE,
-        borderColor: Theme.WHITE
-    },
-    focused_light:{
-        color: Theme.PRIMARY_COLOR,
-        borderColor: Theme.PRIMARY_COLOR
-    },
-    notFocused:{
-    },
-    custom_input: {
-        flex: 1,
-        color:Theme.PRIMARY_COLOR,
-        fontSize:15,
-        paddingVertical:16,
-        alignItems: 'center',
-        justifyContent: 'center'
-    },
-    small_input_container: {
-        flexDirection: 'row',
-        alignItems: 'center',
-        justifyContent: 'space-between'
-    },
-    small_field: {
-        flex:2,
-        marginRight: 5,
-    },
-    password_icon: {
-        padding:8,
-        opacity: .5,
-        alignItems: 'center',
-    },
-    has_comment: {
-        marginBottom:5
-    },
-    input_comment: {
-        width: '100%',
-        fontSize: 11,
-        paddingLeft:5,
-        color: Theme.GRAY,
-        textAlign: 'left'
-    }
-})
+	modalBackground: {
+		flex: 1,
+		justifyContent: "center",
+		alignItems: "center",
+		backgroundColor: "rgba(12,30,52, 0.9)",
+	},
+	modalView: {
+		margin: 20,
+		backgroundColor: Theme.SURFACE_COLOR,
+		justifyContent: "flex-start",
+		borderRadius: 10,
+		padding: 15,
+		alignItems: "center",
+		shadowColor: "#000",
+		shadowOffset: {
+			width: 0,
+			height: 2,
+		},
+		shadowOpacity: 0.25,
+		shadowRadius: 4,
+		width: "90%",
+		height: 420,
+		borderWidth: 1,
+		borderColor: Theme.WHITE,
+	},
+	close_button: {
+		position: "absolute",
+		top: 0,
+		right: 0,
+		padding: 10,
+		zIndex: 10000,
+		fontWeight: Theme.FONT_WEIGHT_HEAVY,
+	},
+	modalHeader: {
+		alignItems: "center",
+		width: "100%",
+		color: Theme.SECONDARY_COLOR,
+	},
+});
 
+const forms = StyleSheet.create({
+	formContainer: {
+		flexDirection: "row",
+		height: 80,
+		marginTop: 40,
+		marginBottom: 20,
+		flex: 1,
+		paddingTop: 10,
+		paddingBottom: 10,
+		paddingLeft: 30,
+		paddingRight: 30,
+		justifyContent: "center",
+		alignItems: "center",
+	},
+	input: {
+		width: "100%",
+		fontSize: 15,
+		lineHeight: 17,
+		backgroundColor: Theme.SURFACE_COLOR,
+		color: Theme.TEXT_ON_SURFACE_COLOR,
+		marginVertical: 15,
+		padding: 10,
+		borderRadius: 5,
+		borderColor: Theme.FAINT,
+		overflow: "hidden",
+		borderWidth: 1,
+		backgroundColor: Theme.SURFACE_COLOR,
+		borderColor: Theme.BORDER_COLOR,
+	},
+	textarea: {
+		height: 120,
+	},
+	search_bar: {
+		height: 48,
+		borderRadius: 25,
+		overflow: "hidden",
+		backgroundColor: Theme.SURFACE_COLOR,
+		color: Theme.SECONDARY_COLOR,
+		borderWidth: 1,
+		borderColor: Theme.PRIMARY_COLOR,
+		paddingLeft: 16,
+	},
+	placeholder_on_dark: {
+		color: "rgba(203, 165, 44, 0.4)",
+	},
+	input_container: {
+		width: "100%",
+		flexDirection: "row",
+		justifyContent: "center",
+		alignItems: "center",
+		paddingLeft: 8,
+		height: 58,
+		borderRadius: 8,
+		marginVertical: 20,
+		borderWidth: 1,
+		backgroundColor: Theme.SURFACE_COLOR,
+		borderColor: Theme.BORDER_COLOR,
+	},
+	input_icon: {
+		color: Theme.SECONDARY_COLOR,
+		padding: 8,
+		alignItems: "center",
+	},
+	focused_dark: {
+		color: Theme.WHITE,
+		borderColor: Theme.WHITE,
+	},
+	focused_light: {
+		color: Theme.PRIMARY_COLOR,
+		borderColor: Theme.PRIMARY_COLOR,
+	},
+	notFocused: {},
+	custom_input: {
+		flex: 1,
+		color: Theme.PRIMARY_COLOR,
+		fontSize: 15,
+		paddingVertical: 16,
+		alignItems: "center",
+		justifyContent: "center",
+	},
+	small_input_container: {
+		flexDirection: "row",
+		alignItems: "center",
+		justifyContent: "space-between",
+	},
+	small_field: {
+		flex: 2,
+		marginRight: 5,
+	},
+	password_icon: {
+		padding: 8,
+		opacity: 0.5,
+		alignItems: "center",
+	},
+	has_comment: {
+		marginBottom: 5,
+	},
+	input_comment: {
+		width: "100%",
+		fontSize: 11,
+		paddingLeft: 5,
+		color: Theme.GRAY,
+		textAlign: "left",
+	},
+});
 
-export {globalStyles, menusStyles, forms, modal}
\ No newline at end of file
+export { globalStyles, menusStyles, forms, modal };

```

armandohimself committed:
Add Small Style Change

armandohimself committed:
Merge branch 'main' into createEvent

armandohimself committed:
Fix Create Event & Template Event Details Screen

Context: 
When you created an event or copied one over from a template, and tried to view it from Your Events tab, the menu courses and items were not showing up. 

Important To Note
1. When you created an event from scratch and it populated in your "Your Events" tab, we would try to grab the menu from the menu templates or the chef's menu's collection. Keep in this in mind as I explain point 2. 
2. When you copied over an event template, we would point to the menu template id and try to grab said menu in the menu templates collection. 

Why this was an issue: 
In either option above, we can can add a flag for the pageName and get our menu courses and items without an issue. 

The problem lies when a copied template is viewed from Your Events because now we don't know where the menu came from originally. 

What I did was:
- Add a flag in the experiences collection to know if the event is[a]Template.
- You can see the menus now on each event when you press on each one in the templates tab.
- Added a flag for only the pricing to show up in the templates if you navigate to the templates tab in both menu and events. 
- Added a tooltip so the user wouldn't be confused about what they see.

armandohimself committed:
Fix Event Template

- When editing an event you can see the location and the menu that was selected before. 
- Added Focus Effect to allow refresh of data.

armandohimself committed:
Merge branch 'main' into fixMenuPrice

armandohimself committed:
Merge branch 'createEvent' into fixMenuPrice
